package.json

```
@@ -6,13 +6,15 @@
   "scripts": {
     "build": "tsc",
     "test": "ava",
-    "lint": "tslint '*(test|src)/**/*.ts'"
+    "lint": "tslint '*(test|src)/**/*.ts'",
+    "prepare": "npm run build"
   },
   "devDependencies": {
     "@types/express": "4.17.2",
     "@types/node": "12.12.11",
     "apollo-boost": "0.4.4",
     "ava": "2.4.0",
+    "axios": "^0.19.0",
```

```suggestion
    "axios": "0.19.0",
```
we try to avoid fix the package versions and avoid to use `^`

---

not needed anymore